script.js

```diff
@@ -0,0 +1,35 @@
+const hello = 'Hello';
+const world = 'World!'
+
+function say() {
+  console.log(`${hello}, ${world}`);
+}
+
+say();
+
+const multiply = (x = 1 ,y = 1) => { console.log(x * y) };
+
+multiply(2,2);
+multiply(5);
+
+function average(...args) {
+  const len = args.length;
+  return args.reduce((prev, curr) => {
```

Comment:
try to write is without {} brackets and without assigning it to new variable

Reply:
what is second argument of reduce function?

Reply:
a current value